src/Mvc/src/Microsoft.AspNetCore.Mvc.Razor/Microsoft.AspNetCore.Mvc.Razor.csproj

```diff
@@ -17,15 +17,15 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.AspNetCore.Mvc.ViewFeatures\Microsoft.AspNetCore.Mvc.ViewFeatures.csproj" />
+    <Reference Include="Microsoft.AspNetCore.Mvc.ViewFeatures" />
 
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.Extensions" Version="$(MicrosoftAspNetCoreMvcRazorExtensionsPackageVersion)" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Runtime" Version="$(MicrosoftAspNetCoreRazorRuntimePackageVersion)" />
     <PackageReference Include="Microsoft.CodeAnalysis.Razor" Version="$(MicrosoftCodeAnalysisRazorPackageVersion)" />
```

Comment:
Had to leave this as explicit package reference

Reply:
+1. Can you share what doesn't work if you change to `Reference`?

Reply:
Because I get errors like these in some of the functional tests 
```
k3dz3hwj.fsu(32,59): error CS1705: Assembly 'Microsoft.AspNetCore.Razor.Runtime' with identity 'Microsoft.AspNetCore.Razor.Runtime, Version=2.1.2.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' uses 'Microsoft.AspNetCore.Razor, Version=2.1.2.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' which has a higher version than referenced assembly 'Microsoft.AspNetCore.Razor' with identity 'Microsoft.AspNetCore.Razor, Version=2.1.1.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'
```

Reply:
Can you collect a binlog of the test project? It appears the test project is getting the wrong versions of dependencies, but I'm not sure why. Possibly due to Directory.Build.targets/props imports.

Reply:
Discussed offline. Filed https://github.com/aspnet/AspNetCore/issues/6070 to clean this up later